development/mimir-microservices-mode/docker-compose.jsonnet

```diff
@@ -33,6 +33,7 @@ std.manifestYamlDoc({
 
     // If true, a query-tee instance with a single backend is started.
     enable_query_tee: false,
+    enable_ha_tracker: true,
```

Comment:
Can you include some docs here (some of the info from the description of this PR would be good)?